### PR TITLE
Prospector

### PR DESCRIPTION
**Pull request by AndreBiedenkapp**

### CODE REVIEW

File: .prospector.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+pep8:
+  disable:
+    - N806              # ignore pep8 naming (func arg should be lowercase i.e. X_pred)
+  options:
+    max-line-length: 120
+
+pylint:
+  disable:
+    - C0326              # ignore pylint bad-whitespace  (func arguments)
```
Comment (mfeurer) [inline .prospector.yaml]: Is there a reason to not check this one?

Comment (AndreBiedenkapp) [reply .prospector.yaml]: We have a bunch of argument in functions like https://github.com/automl/SMAC3/blob/5775eaf234730c8aaaa06e46d0eb6e0a8348929e/smac/epm/rf_with_instances.py#L117
which all would create issues.

Comment (AndreBiedenkapp) [reply .prospector.yaml]: So either we replace all these function arguments that have uppercase letters with lower case ones or we ignore this one rule

Comment (mfeurer) [reply .prospector.yaml]: I actually meant `C0326              # ignore pylint bad-whitespace  (func arguments)`.

Comment (AndreBiedenkapp) [reply .prospector.yaml]: Oh sorry my bad. I confused myself with the func arguments.
This should be removed. I accidentally left when testing if I configure prospector correctly.

**Pull request merged**